### PR TITLE
created a table for a dashboard 23#

### PR DESCRIPTION
**Pull request by KristinaDudnyk**
On the server side, in  folder data.sql was created a dashboard table with all required  columns
https://drawsql.app/teams/aas-4/diagrams/copy-of-deja-review

### CODE REVIEW

**Review by Abubakar-Meigag — COMMENTED** [review]
Hey Kristina, there are only two comments that need clarification. Also, I checked in PostgreSQL and didn't see your table. Could you please create it there?

let me know if you need any help even connecting to postgres 

Thank you Kristina

File: server/database/data.sql [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
+	('Joseph', 'Mwanza' 'joseph@gmail.com');
 
+-- Dashboard table
+CREATE TABLE dashboard (
+	id SERIAL PRIMARY KEY,
+	topic_id INT,
+	user_id INT,
+	task_status ENUM('Not Started', 'In Progress'),
+	due_date timestamp,
+	reviews_remaining INT DEFAULT 5
```
[the remark this answers is not in the document]
Comment (KristinaDudnyk) [reply server/database/data.sql:30]: Hi Beko, 
So do I need to change the number to 4 ? Or do I need to change it in a different way?

**Review by ElenaBarker — COMMENTED** [review]
Hi Kristina, 
I just wanted to add one small thing: in our final database schema, the "dashboard" table has the name "topic_tracker". Can you rename it, please? Thank you

**Review by Abubakar-Meigag — APPROVED** [review]
review done

**Pull request merged**